Stripe/PublicHeaders/UI/STPPaymentCardTextField.h

```diff
@@ -220,6 +220,11 @@
  */
 @property(nonatomic, readonly, nullable) STPCardParams *card;
```

Comment:
actually, forgot there was a reason I did this – we probably don't want to change the `card` on the legacy `PTKView : STPPaymentCardTextField` interface to `readwrite`, and we can't update this property without also updating that subclass.